exercises/shared/.docs/help.md

```diff
@@ -4,3 +4,274 @@ To get help if you're having trouble, you can use one of the following resources
 
 - [/r/WebAssembly](https://www.reddit.com/r/WebAssembly/) is the WebAssembly subreddit.
 - [Github issue tracker](https://github.com/exercism/wasm/issues) is where we track our development and maintenance of Javascript exercises in exercism. But if none of the above links help you, feel free to post an issue here.
+
+# How to Debug
```

Comment:
The headings should increase one level:

```suggestion
## How to Debug
```

This also applies to the other headings.